packages/repository/src/model.ts

```diff
@@ -106,6 +106,15 @@ export class ModelDefinition {
     const definition = (definitionOrType as PropertyDefinition).type
       ? (definitionOrType as PropertyDefinition)
       : {type: definitionOrType};
+
+    if (
```

Comment:
what if I have:
```ts
@property({
  id: true,
  generated: true,
  useDefaultIdType: true
})
id: string;
```
Would it overwrite the value of `useDefaultIdType` above?

Reply:
I added a check `&&
      definition.useDefaultIdType === undefined`

Reply:
Good approach, I have seen that you have already changed it so I will give this revision as resolved